CONTRIBUTING.md

```diff
@@ -86,18 +86,24 @@ While you're working away in your branch it's quite possible that your upstream/
-1. `git push origin master` - (optional) this this makes sure your remote master is up to date
-1. if you previously pushed your branch to your origin, you need to force push the rebased branch - `git push origin myBranch -f`
+1. `git push origin master` - (optional) this makes sure your remote master is up to date if you previously pushed your branch to your origin, you need to force push the rebased branch - `git push origin myBranch -f`
 
 This ensures that your history is "clean" i.e. you have one branch off from master followed by your changes in a straight line. Failing to do this ends up with several "messy" merges in your history, which we don't want. This is the reason why you should always work in a branch and you should never be working in, or sending pull requests from, master.
 
 If you're working on a long running feature then you may want to do this quite often, rather than run the risk of potential merge issues further down the line.
 
 ## Sending a Pull Request
 
-While working on your feature you may well create several branches, which is fine, but before you send a pull request you should ensure that you have rebased back to a single "feature branch". We care about your commits and we care about your feature branch but we don't care about how many or which branches you created while you were working on it :-)
+While working on your feature you may well create several branches, which is fine, but before you send a pull request you should ensure that you have rebased back to a single ["feature branch"](https://martinfowler.com/bliki/FeatureBranch.html). We care about your commits and we care about your feature branch but we don't care about how many or which branches you created while you were working on it. :-)
 
 When you're ready to go you should confirm that you are up to date and rebased with upstream/master (see "Handling Updates from upstream/master" above) and then:
 
 1. `git push origin myBranch`
-1. Send a [pull request](https://help.github.com/articles/using-pull-requests) on GitHub. Make sure you select `FakeItEasy/FakeItEasy` as the *base repo* and `master` as the *base branch*. Select your fork as the *head repo* and your branch as the *head branch*. The pull request should include a description starting with "Fixes #123." (using the real issue number, of course) if it fixes an issue. If there's no issue, be sure to clearly explain the intent of the change.
+1. Send a [pull request](https://help.github.com/articles/using-pull-requests) on GitHub. Make sure to create the pull request using the information from the table below. The pull request should include a description starting with "Fixes #123." (using the real issue number, of course) if it fixes an issue. If there's no issue, be sure to clearly explain the intent of the change.
+
+|Field | Value     |
+|---- | ------------ |
+|base fork      | **FakeItEasy/FakeItEasy** *(always)*      |
+|base branch    | **master** *(always)* |

```

Comment:
Note that this might change soon, as we intend to switch to [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/). It's correct for the current process, though.